src/Application.vala

```diff
@@ -186,10 +182,6 @@ public class Files.Application : Gtk.Application {
         }
 
         /* Handle arguments */
-        if (debug) {
-            Granite.Services.Logger.DisplayLevel = Granite.Services.LogLevel.DEBUG;
-        }
```

Comment:
Either remove the debug argument or do:
```vala
if (debug) {
    GLib.Environment.set_variable ("G_MESSAGES_DEBUG", "all", false);
}

Reply:
Good point - I was a bit trigger-happy there.